arcee/api.py

```diff
@@ -48,28 +70,66 @@ def upload_docs(context: str, docs: List[Dict[str, str]]) -> Dict[str, str]:
     data = {"context_name": context, "documents": doc_list}
     return make_request("post", Route.contexts, data)
 
+def start_pretraining(pretraining_name: str, corpus: str, base_model: str) -> None:
+    """
+    Start pretraining a model
+
+    Args:
+        pretraining_name (str): The name of the pretraining job
+        corpus (str): The name of the corpus to use
+        base_model (str): The name of the base model to use
+    """
+
+    data = {"pretraining_name": pretraining_name, "corpus_name": corpus, "base_model": base_model}
+
+    return make_request("post", Route.pretraining+"/startTraining", data)
+
+def start_alignment(alignment_name: str, qa_set: str, pretrained_model: str) -> None:
+    """
+    Start alignment of a model
+
+    Args:
+        alignment_name (str): The name of the alignment job
+        qa_set (str): The name of the QA set to use
+        pretrained_model (str): The name of the pretrained model to use
+    """
 
-def train_dalm(
-    name: str, context: Optional[str] = None, instructions: Optional[str] = None, generator: str = "Command"
-) -> None:
-    data = {"name": name, "context": context, "instructions": instructions, "generator": generator}
+    data = {"alignment_name": alignment_name, "qa_set_name": qa_set, "pretrained_model": pretrained_model}
+
+    return make_request("post", Route.alignment+"/startAlignment", data)
+
+def start_retriever_training(name: str, context: str):
```

Comment:
I'm assuming this function will ultimately be a wrapper around the [train_retriever](https://github.com/arcee-ai/arcee-train/blob/main/src/train_retriever.py) code.  

But we also currently have this [train_generator](https://github.com/arcee-ai/arcee-train/blob/main/src/train_generator.py) code, which builds a reading comprehension dataset and fine-tune the generator on that dataset.  

I'm assuming `train_generator.py` will _not_ be included in this new training workflow, but I just want to confirm that.